web/source/osk/oskManager.ts

```diff
@@ -261,8 +261,14 @@ namespace com.keyman.osk {
       }
 
       // Correct the classname for the (inner) OSK frame (Build 360)
+      var kbdID: string = (activeKeyboard ? activeKeyboard.id.replace('Keyboard_','') : '');
+      if(keymanweb.isEmbedded && kbdID.indexOf('::') != -1) {
+        // De-namespaces the ID for use with CSS classes.
+        // Assumes that keyboard IDs may not contain the ':' symbol.
```

Comment:
Keyboard IDs have to be valid Javascript identifiers. So punctuation is not possible. Also, we restrict the names to `[A-Za-z0-9_]+` anyway (and disallow digits as first letter)

Reply:
That's what I thought.  Always best to make the assumptions explicit just in case, though.